### PR TITLE
Adding ReleaseLevel concept, supporting in Java

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
The "unset" case is present in the no_path_templates service.

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/java_main_library.baseline [hunk 1]
```diff
@@ -153,7 +153,6 @@ import javax.annotation.Generated;
  * </pre>
  */
 @Generated("by GAPIC")
-@ExperimentalApi
```
Comment (michaelbausor) [inline src/test/java/com/google/api/codegen/testdata/java_main_library.baseline]: Did you intent to test the ALPHA case, when this would still show up?

**Comment by michaelbausor** [comment]
Also, where you intending to set ALPHA on all APIs? Otherwise you might want the unset case to include experimentalApi for Java

File: src/test/java/com/google/api/codegen/testdata/java_main_library.baseline (hunk 1 above)
Comment (garrettjonesgoogle) [inline src/test/java/com/google/api/codegen/testdata/java_main_library.baseline]: I didn't want to create two whole new services (for ALPHA and GA), that would be a bit of overhead.

**Comment by garrettjonesgoogle** [comment]
I was going to leave APIs unset for now. The unset case *does* include ExperimentalApi; notice that the no_path_templates service is untouched in this PR.

**Pull request merged**